nodejs/src/table.rs

```diff
@@ -108,7 +108,7 @@ impl Table {
         })
     }
 
-    #[napi]
+    #[napi(catch_unwind)]
```

Comment:
A bit unfortunate we have to repeat this.

Reply:
yeah, it'd be nice if there was a way to enable this by default.